docs/source/models/gbd2017/causes/causes.rst

```diff
@@ -108,13 +108,41 @@ The Appendix for the GBD 2017 Cause of Death manuscript is also available and Op
 Modeling non-fatal outcomes (Outline)
 ---------------------------
 
-- Figure of process
 .. image:: gbd_nonfatal_flowchart_sim_team.jpg
```

Comment:
Currently, it notes under "Final results in GBD-Compare" that one can pull deaths, incidence, prev, YLDs, YLLs, DALYs. Might be nice to have another arrow pointing into that final green rectangle (from 'fatal modeling pipeline' or something similar), to show that half of those aren't a product of everything else visualized here--or in some other way differentiate those

The boxes "epi database" and "modeling and computational tools" might be clearer as labels for those sections if they don't have their own bubbles? But that's definitely up to personal preference!

Reply:
I would also consider adding (i don't know if you or if james or someone else mocked up the outline?) 'epi database' to the list of things that you're going to define